tools/metadata_merger/test-data/aplcms_converted-test.csv

```diff
@@ -0,0 +1,7 @@
+sample_name,100.01124231464279_316.6324148139556,100.0140326116781_370.3911481191968,100.03941952410403_150.6579990293048,100.07581095178341_53.375130879368484
+VT_160120_002,29239096.399301223,40207085.08482562,3031215.6621509064,289190890.42614394
+VT_160120_004,9373024.38821382,39892591.38643598,1156692.2274786886,314872910.16817033
+VT_160120_006,11575595.878938954,41211883.00982785,1188030.695089618,308427518.55757684
+VT_160120_008,26277808.30956555,45866485.05309503,2255800.536242036,119045892.73239136
+dataset_4692,24658120.59201486,38960230.26046665,4663513.710815267,144921892.4161568
```

Comment:
What do these lines starting with `dataset_` mean?

Reply:
These are a few samples that don't have corresponding metadata and should not be included in the output. Just to check if the join is performed correctly.